src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Promotion.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource class="%sylius.model.promotion.class%" shortName="Promotion">
+        <attribute name="normalization_context">
+            <attribute name="groups">
+                <attribute>admin:promotion:read</attribute>
+            </attribute>
+        </attribute>
+
+        <attribute name="validation_groups">sylius</attribute>
+
+        <collectionOperations>
+            <collectionOperation name="admin_get">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">admin/promotions</attribute>
```

Comment:
You could add `<attribute name="route_prefix">admin</attribute>` as all of the endpoints are for admin